stroeer/core/v1/article.proto

```diff
@@ -411,23 +411,28 @@ message Metadata {
 
   /**
   * ## `enum State`
-  * State of the article in the content management system.
+  *
+  * State of the item ([`Article`](article.html), [`Element`](element.html))
+  * in the content management system. The `state` in combination with
+  * [start_time](#start_time) and [end_time](#end_time) determines whether or not
+  * this item should be rendered; this must be respected by all consumers especially
+  * when content is duplicated or cached.
+  *
+  * The terms `deleted` (articles) and `archived` (media lib) are interchangeable/synonyms.
+  * This API combines those two into `State.DELETED`.
+  *
   * ```protobuf
-  *   enum State {
-  *   // Not specified.
+  * enum State {
   *   STATE_UNSPECIFIED = 0;
-  *
-  *   // A published article.
   *   PUBLISHED = 1;
+  *   DELETED = 2;
   * }
   * ```
   */
   enum State {
-    // Not specified.
     STATE_UNSPECIFIED = 0;
-
-    // A published article.
     PUBLISHED = 1;
+    DELETED = 2;
```

Comment:
The cms api responds with

```json
{
"state": "archived"
}
```

for articles which have been deleted (label of the CMS button). Guess combining this into `State.DELETED` is fine.